src/map/status.hpp

```diff
@@ -161,6 +161,7 @@ enum sc_type : int16 {
 	SC_BLIND,
 	SC_BLEEDING,
 	SC_DPOISON, //10
+	SC_STONEWAIT,
 	SC_COMMON_MAX = 10, // end
```

Comment:
Should STONEWAIT be included in the common status or not?

Reply:
I only thought it deemed appropriate because we do have some logic checks in source for SC_COMMON*.

Reply:
By the way shouldn't the value of SC_COMMON_MAX change to 11 then?